docs/cog_guides/filter.rst

```diff
@@ -0,0 +1,303 @@
+.. _filter:
+
+======
+Filter
+======
+
+This is the cog guide for the filter cog. You will
+find detailed docs about usage and commands.
+
+``[p]`` is considered as your prefix.
+
+.. note:: To use this cog, load it by typing this::
+
+        [p]load filter
+
+.. _filter-usage:
+
+-----
+Usage
+-----
+
+This cog is designed for "filtering" unwanted words and phrases from a guild.
+
+It provides tools to manage a list of words or sentences, and to customize automatic actions to be taken against users who use those words in channels or in their name/nickname.
+
+This can be used to prevent inappropriate language, off-topic discussions, invite links, and more.
+
+
+.. _filter-commands:
+
+--------
+Commands
+--------
+
+.. _filter-command-filter:
+
+^^^^^^
+filter
+^^^^^^
+
+.. note:: |mod-lock|
+
+**Syntax**
+
+.. code-block:: none
+
+    [p]filter 
+
+**Description**
+
+Base command to add or remove words from server filter.
```

Comment:
```suggestion
Base command to add or remove words from the server filter.
```
I'm also not a fan of "*add* or *remove* words *from*", but there isn't a good way to make this sound better besides being overly verbose ("add words to or remove words from"). Unless you can think of a better wording I'm fine leaving that how it is.